src/view/com/util/fab/FABInner.tsx

```diff
@@ -18,6 +20,26 @@ export function FABInner({testID, icon, ...props}: FABProps) {
   const insets = useSafeAreaInsets()
   const {isMobile, isTablet} = useWebMediaQueries()
   const {fabMinimalShellTransform} = useMinimalShellMode()
+  const styles = useStyles(React.useMemo(() => ({
+    sizeRegular: {
+      width: 60,
+      height: 60,
+      radius: 'round',
+    },
+    sizeLarge: {
+      width: 70,
+      height: 70,
+      radius: 'round',
+    },
+    outer: {
+      position: 'absolute',
+      zIndex: 1,
+    },
+    inner: {
+      justifyContent: 'center',
+      alignItems: 'center',
+    },
+  }), []))
```

Comment:
Could definitely do this better with some better button/pressable primitives, but I did this just to show that we _can_ just copy pasta in the defs from `StyleSheet` if there are cases where we must.

For entirely static styles in small specific components like this — especially in perf-intensive situations — maybe StyleSheet is still the way to go.